src/components/PagingControl.tsx

```diff
@@ -78,7 +79,9 @@ export default function withPagingControl<T, T1>(
     render(): React.ReactElement {
       const items = this.state.items
       if (!items) {
-        return <div/>
+        return <Alert variant="primary">
+          <Alert.Heading>No Items Were Found!</Alert.Heading>
```

Comment:
The `<Alert.Heading>` makes this message a little too punchy. Here's a screen grab:

![Screen Shot 2022-03-02 at 4 00 06 PM](https://user-images.githubusercontent.com/1821014/156448930-34c53838-69af-418e-bc08-2ae82508eecb.png)

Possible that we can just use `<Alert>` and no `<Alert.Heading>`?

Similarly, the caps and exclamation make this message sound exciting. lol. If we can dial it back to "No items were found." that is probably a better tone.